tests/__init__.py

```diff
@@ -36,6 +39,12 @@
 _LOADER = botocore.loaders.Loader()
 
 
+def requires_memory_collection(cls):
```

Comment:
Not opposed to renaming it.  Technically, this is skipping the test if the system doesn't support memory collection, but that would be pretty long: `skip_unless_has_memory_collection`.  You ok with that?